docs/Concepts/Privacy/PrivateTransactionManager.md

```diff
@@ -6,14 +6,47 @@ private transactions.
 The Private Transaction Manager is a separate component that is concerned with the storing and distribution
 of encrypted private transaction data between recipients of a private transaction.
 
+The Private Transaction Manager has two distinct components:
+
+* Transaction Manager
+* Enclave
+
 To enable private transactions, use the `PRIVATE_CONFIG` environment variable when starting a GoQuorum
-node to provide the node with the path to the Privacy Manager's `.ipc` socket, e.g.:
+node to provide the path to the Privacy Manager's `.ipc` socket.
+
+Use the `PRIVATE_CONFIG` environment variable either with a direct path to the IPC socket or with a path to a configuration file as explained in the following sections. 
+
+## Direct IPC connection configuration
+
+This is the basic method for specifying the IPC socket:
```

Comment:
I would have preferred to leave this saying that this is the "normal" method as we want to discourage folks from using the more complex method of having a TOML file unless it's really necessary.

Reply:
Why? Basic means you fulfil basic needs which is the usual use case, but normal implies that the other one is not normal and thus a bit hacky.